packages/flutter_tools/test/general.shard/ios/ios_deploy_test.dart

```diff
@@ -147,6 +147,33 @@ void main () {
         expect(logger.traceText, contains('* thread #1'));
       });
 
+      testWithoutContext('debugger attached and stop failed', () async {
+        final StreamController<List<int>> stdin = StreamController<List<int>>();
+        final FakeProcessManager processManager = FakeProcessManager.list(<FakeCommand>[
+          FakeCommand(
+            command: const <String>['ios-deploy'],
+            stdout: '(lldb)     run\r\nsuccess\r\nsuccess\r\nprocess signal SIGSTOP\r\n\r\nerror: Failed to send signal 17: failed to send signal 17',
```

Comment:
Out of curiosity, would we actually expect to see `'\r'` here?

Reply:
Oh good question, I'm not sure. I just grabbed that from the test above. I'll try and figure out

Reply:
not worth investigating, just wondering if there was a legit reason.

Reply:
I just happened to come across the answer to this: https://github.com/flutter/flutter/blob/4f6f1887c5e0b610a906c1b3eb5f01a8d2a555bc/packages/flutter_tools/lib/src/ios/ios_deploy.dart#L477-L480